### DISCUSSION TITLE
Week 3

### DISCUSSION THREAD

**Discussion by Stev3eM** [SE-TINF22B6/CookHub#39] [Show and tell]
Hello backpackers,

This is our weekly discussion-blog to keep you up to date within our project progression.

For the third week we have some updates for :

✅ Created our srs.md file with Software Requirements Specification
https://github.com/SE-TINF22B6/CookHub/blob/main/srs.md

✅ Created UML diagram via draw.io
<img width="300" alt="image" src="https://github.com/SE-TINF22B6/CookHub/assets/113697513/4623908f-6ebc-4ca2-8126-8769efab6a1a">

✅ Created login page via figma
<img width="300" alt="image" src="https://github.com/SE-TINF22B6/CookHub/assets/113697513/13b1630f-871c-4630-96b0-4e31cd8bc50d">

✅ Created profile page via figma
<img width="300" alt="image" src="https://github.com/SE-TINF22B6/CookHub/assets/113697513/ac9bf059-7373-41b5-a60e-8050ce79b08c">

✅ Created settings page via figma
<img width="300" alt="image" src="https://github.com/SE-TINF22B6/CookHub/assets/113697513/44140964-f86b-40cb-845a-27e0cbe30951">

<img width="50" alt="image" src="https://github.com/SE-TINF22B6/CookHub/assets/113697513/cdbd0688-3eb9-46fa-a608-baf2efbaba88">  Your CookHub Team

**Comment by JonasEbbinghaus25** [comment]
Your blog shows that you accomplished a lot during the week. The comprehensive Software Requirements Specification sets a strong foundation for your development journey. The detailed overview, functional and non-functional requirements, and the technical constraints are crucial for ensuring the project's success. It's worth mentioning the appealing mockups you've created in Figma for the GUI. These visual representations not only look great but also provide a clear direction for your user interface development. The effort put into the design and layout is evident and promising.

Additionally, your UML diagram is commendably straightforward, making it easy to understand the major use cases of your application. The logo looks great and fits well with the project.

**Comment by TadaXD20** [comment]
Great overview of the finished tasks! It includes all the needed points and is not overwhelming. Also the use case diagram is makes it easy to understand all the features of the app. The mockups encourage to try out the app.

Tanina from Team Plapy

**Comment by sfrancisw** [comment]
Great job on the progress you've made! The srs looks exeptionally well made and has all the information that it needs! The figma pages look very nice and modern. Looking forward to see them implemented and in action!

**Comment by ghost** [comment]
We like the UI design and the structure of the model is very exact and clear. Great job im excited for updates.
